src/main/java/edu/kis/powp/jobs2d/commands/ComplexCommand.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.powp.jobs2d.commands;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.DriverCommand;
+
+import java.util.ArrayList;
+
+public class ComplexCommand extends DriverCommand {
+    public ArrayList<DriverCommand> list = new ArrayList<>();
```

Comment:
Should be private!
List type (attribute and in the constructor) would be sufficient and would not limit the implementation.